unix/x0vncserver/x0vncserver.cxx

```diff
@@ -260,6 +260,10 @@ int main(int argc, char** argv)
 
   Configuration::enableServerParams();
 
+  // Disable configuration parameters which we do not support
+  Configuration::setParam("AcceptSetDesktopSize", "0");
```

Comment:
This is unnecessary as the default implementation will deny everything anyway.

Reply:
Yes, but without this code, AcceptSetDesktopSize will be listed in --help output, which is confusing.

Reply:
I meant specifically the `setParam()` call. The `removeParam()` call does indeed do useful things.